src/plugins/lua/echotest.lua

```diff
@@ -7,14 +7,14 @@ json = require('json')
 -- We also import our own SDP helper utilities: you may have better ones
 sdp = require('janus-sdp')
 -- Let's also use our ugly stdout logger just for the fun of it: to add
--- some color to the text we use the ansicolors library
--- (https://github.com/kikito/ansicolors.lua)
-colors = require "ansicolors"
+-- some color to the text we use the lua-trm library
+-- (https://github.com/hoelzro/lua-term)
+colors = require "term.colors"
```

Comment:
I've just noticed we're using a different syntax for this require, could you align it to the others?